doc/examples/applications/plot_solidification_tracking.py

```diff
@@ -0,0 +1,155 @@
+
+image_sequence = nickel_solidification()
+
+y0, y1, x0, x1 = 0, 180, 100, 330
+
+image_sequence = image_sequence[:, y0:y1, x0:x1]
+
+print(f'shape: {image_sequence.shape}')
+
+#####################################################################
+# The dataset is a 2D image stack with 11 frames (time points).
+
+fig = px.imshow(
+    image_sequence,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Compute image deltas
+# ====================
+# Let us apply a Gaussian low-pass filter to the images in order to smooth
+# the images and reduce noise.
+# Next, we compute the image deltas, i.e., the sequence of differences
+# between two consecutive frames. To do this, we subtract ``image_sequence``
+# from itself, but offset by one frame so that the subtracted images are 
+# one frame behind in time.
+
+images_smoothed = filters.gaussian(image_sequence)
+image_deltas = images_smoothed[1:, :, :] - images_smoothed[:-1, :, :]
+
+fig = px.imshow(
+    image_deltas,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Clip lowest and highest intensities
+# ===================================
+# We now calculate the 5th and 95th percentile intensities of ``image_deltas``:
+# We want to clip the intensities which lie below the 5th percentile
+# intensity and above the 95th percentile intensity, while also rescaling 
+# the intensity values to [0, 1]. 
+
+p_low, p_high = np.percentile(image_deltas, [5, 95])
+clipped = image_deltas - p_low
+clipped[clipped < 0.0] = 0.0
+clipped = clipped / p_high
+clipped[clipped > 1.0] = 1.0
+
+fig = px.imshow(
+    clipped,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Invert and denoise
+# ==================
+# Next, we invert the ``clipped`` images so the regions of highest intensity
+# will correspond to the region we are interested in tracking (i.e., the 
+# S-L interface). We now apply a total variation denoising filter to reduce 
+# noise beyond the interface.
+
+inverted = 1 - clipped
+denoised = restoration.denoise_tv_chambolle(inverted, weight=0.15)
+
+fig = px.imshow(
+    denoised,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig) 
+
+#####################################################################
+# Binarize
+# ========
+# Our next step is to create binary images, splitting each image 
+# into a foreground and a background: We want the S-L interface 
+# to be the most prominent feature in the foreground of each binary image,
+# so that it can eventually be separated from the rest of the image.
```

Comment:
Maybe the phrasing with 'each' in this paragraph is a little misleading: It may sound as if each (2D) image in the sequence would be processed separately (typically, computing a threshold value for each image).